v2/api-validator/src/client/generated/models/CollateralWithdrawalTransactionIntentResponse.ts

```diff
@@ -0,0 +1,19 @@
+/* generated using openapi-typescript-codegen -- do no edit */
+/* istanbul ignore file */
+/* tslint:disable */
+/* eslint-disable */
+
+import type { ApprovalRequest } from './ApprovalRequest';
+import type { CollateralTransactionIntentStatus } from './CollateralTransactionIntentStatus';
+
+export type CollateralWithdrawalTransactionIntentResponse = {
```

Comment:
We said we're currently not going to have an intent response per endpoint family